LICENSE-3rdparty.csv

```diff
@@ -1,16 +1,22 @@
 Component,Origin,License,Copyright
+PyJWT,PyPI,MIT,Copyright (c) 2015 José Padilla
 PyJWT,PyPI,MIT,Copyright (c) 2015-2022 José Padilla
 PyMySQL,PyPI,MIT,"Copyright (c) 2010, 2013 PyMySQL contributors"
 PySocks,PyPI,BSD-3-Clause,Copyright 2006 Dan-Haim. All rights reserved.
 PyYAML,PyPI,MIT,Copyright (c) 2017-2021 Ingy döt Net
 aerospike,PyPI,Apache-2.0,"Copyright Aerospike, Inc."
 aws-requests-auth,PyPI,BSD-3-Clause,Copyright (c) David Muller.
+azure-identity,PyPI,MIT,Copyright (c) Microsoft Corporation
 azure-identity,PyPI,MIT,Copyright (c) Microsoft Corporation.
+beautifulsoup4,PyPI,MIT,Copyright (c) 2004-2017 Leonard Richardson
 beautifulsoup4,PyPI,MIT,Copyright (c) Leonard Richardson
 binary,PyPI,Apache-2.0,Copyright 2018 Ofek Lev
 binary,PyPI,MIT,Copyright 2018 Ofek Lev
 boto3,PyPI,Apache-2.0,"Copyright 2013-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved."
+boto3,PyPI,Apache-2.0,Copyright 2014 Amazon Web Services
 botocore,PyPI,Apache-2.0,"Copyright 2012-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved."
+botocore,PyPI,Apache-2.0,Copyright 2012 Amazon Web Services
+cachetools,PyPI,MIT,Copyright (c) 2014-2019 Thomas Kemmer
```

Comment:
I'm mildly curious as to why we're suddenly picking up older copyright notices for several libs that were already here - is this because we added old versions as a py2 dependencies?